src/cli/cluster/cluster_manager.js

```diff
@@ -83,8 +80,8 @@ module.exports = class ClusterManager {
   }
 
   setupWatching(extraPaths) {
-    const chokidar = require('chokidar');
-    const fromRoot = require('../../utils/from_root');
+    const chokidar = require('chokidar'); // kibana-jscodeshift-ignore
```

Comment:
It's a signal to the jscodeshift scripts that the require statement on that line needs to stay on that line, I planned to remove them before submitting the pr